SECP-service/src/main/resources/assets/app/views/home/home.html

```diff
@@ -98,12 +98,12 @@ <h2>We
         </h2>
         <ul class="list-inline list-social">
```

Comment:
Since we have no social media accounts do we need this?

Reply:
I have an open branch now that takes care of that.